src/store/store.ts

```diff
@@ -158,17 +142,17 @@ export const useStore = defineStore('store', {
 
     /** The account folio number. */
     getFolioNumber (): string {
-      return this.getTombstone.folioNumber
```

Comment:
I got rid of `getTombstone` as it was only used in this file and, in my opinion, complicated the understanding of where the data came from -- compare with new code which is now consistent across getters >>>